deepchem/trans/tests/test_transformers.py

```diff
@@ -482,3 +485,42 @@ def test_featurization_transformer(self):
 
     self.assertEqual(new_train.y.shape, train.y.shape)
     self.assertEqual(new_train.X.shape[-1], fp_size)
+
+  def test_data_transformers(self):
+    mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
+    # extracting validation set of MNIST for testing the DataTransforms
+    valid = dc.data.NumpyDataset(mnist.validation.images,
+                                 mnist.validation.labels)
+    # extract only the images (no need of the labels)
+    data = (valid.X)[0]
+    # reshaping the vector to image
+    data = np.reshape(data, (28, 28))
+
+    dt = DataTransforms(data)
+
+    # Check Blurring
+    blurred = dt.gaussian_blur(sigma=1.5)
+    check_blur = scipy.ndimage.gaussian_filter(data, 1.5)
+    assert np.allclose(check_blur, blurred)
+
+    # Check rotation
+    angles = [0, 5, 10, 90]
```

Comment:
can't you do a check for say -90 = 270 here?

Reply:
It might not be exactly the same in terms of pixel values. So what test should I use instead of `np.allclose()`?

Reply:
Done. It worked with `np.allclose()` itself.